olm/src/jvmTest/kotlin/io/github/matrixkt/olm/runTest.kt

```diff
@@ -0,0 +1,6 @@
+package io.github.matrixkt.olm
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.runBlocking
+
+actual fun BaseTest.runTest(block: () -> Unit) = runBlocking { block() }
```

Comment:
Same here. (Don't forget the imports)

```suggestion
actual fun BaseTest.runTest(block: () -> Unit) { block() }
```